.gitignore

```diff
@@ -8,4 +8,6 @@ pie.bat
 demo-local.env
 demo-local-cn.env
 
-/var/
\ No newline at end of file
+/var/
+.venv/
```

Comment:
Why do you have a venv? This should all be running in docker... 😁

Reply:
pie.py didn't like my house python ;)